reVX/setbacks/water_setbacks.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""
+Compute setbacks exclusions
+"""
+from reVX.setbacks.base import BaseSetbacks, features_clipped_to_county
+
+
+class WaterSetbacks(BaseSetbacks):
+    """Water setbacks. """
+
+    @staticmethod
+    def _feature_filter(features, cnty):
+        """Filter the features given a county."""
+        return features_clipped_to_county(features, cnty)
+
+    def _parse_regulations(self, regulations_fpath):
+        """
+        Parse water regulations, reduce table to just water features
+
+        Parameters
+        ----------
+        regulations_fpath : str
+            Path to parcel regulations .csv file
+
+        Returns
+        -------
+        regulations : pandas.DataFrame
+            Parcel regulations table
+        """
+        regulations = super()._parse_regulations(regulations_fpath)
+
+        feats = regulations['Feature Type'].apply(str.strip).apply(str.lower)
+        regulations = regulations.loc[feats == 'water']
```

Comment:
Nice, inheritance is pretty powerful!

Reply:
Minor and late to the party: `feats = regulations['Feature Type'].str.strip().str.lower()` is an alternative to `apply()`

Reply:
Oh dang, didn't know about that shorthand. Will keep it in mind moving forward!